### ISSUE TITLE
Change cursor focus to creator name inputs after creator search

### ISSUE THREAD

**Issue by srerickson** [psu-libraries/scholarsphere#893]
When there is an unidentified creator and the name is entered into the edit field, when the user hits enter, the focus should be on display name. Instead, the focus is right below the display name edit field. 

Question for Michelle - Do we need more text to explain that we need additional info here for the creator whose name was just entered?  It may be confusing to jump right into display name.

**Comment by awead** [comment]
New behavior in #1003 removes the need for this ticket.
